py3_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python3
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy
+
+RUN pip install scikit-learn==0.17
```

Comment:
yep: http://askubuntu.com/questions/509623/installation-procedure-for-numpy-and-other-python-3-4-packages-on-ubuntu-14-04